osCam/templates/core/home.html

```diff
@@ -44,7 +44,7 @@ <h2>cameras</h2>
                         <!-- cameras -->
                         {% if user.is_authenticated %}
                         <div class="container-fluid">
-                        <img src="{% url 'feed' %}">
+                        <img src="{% url 'feed' %}" style="border: 1px solid #eee">
```

Comment:
there's some styling still floating around the html but our goal is all css in dedicated files under "static/____.css"

Reply:
okay, I'll make a new file for the css